eng/build.yml

```diff
@@ -55,7 +57,7 @@ jobs:
     ${{ if eq(parameters.osGroup, 'Linux') }}:
       container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-359e48e-20200313130914
 
-    ${{ if eq(parameters.osGroup, 'Linux-musl') }}:
+    ${{ if eq(parameters.osGroup, 'Linux_Musl') }}:
```

Comment:
On this change, can you update the vmImage to ubuntu-18.04 or ubuntu-latest? 16.04 is EoL pretty soon (not the buildpool part).